usagov/usagov-2021
# USAGOV-1599 & USAGOV-1600 Homepage changes and primary cards

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1599
https://cm-jira.usa.gov/browse/USAGOV-1600

## Description

- Fix the title of the all topics to align with the cards
- Change the drop shadow of the all topics cards and the cards in primary pages to `box-shadow: 1px 1px 7px #aaa;`
- Change the border of the all topics cards to match the cards in the primary pages
- Clean the code and unify the style shared by the cards on the primary pages and the cards in the all topics section of the homepage.
- Increase the width of the All topics cards in mobile and tablet view
- Centered the Welcome and How do I boxes in mobile views

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


1. Go to the homepage and check that it looks like this (Check the alignment and width of the cards with the other things)
2. Check that the border (check the hover as well) of the all topics matches the border of the cards in the primary pages
<img width="299" alt="image" src="https://github.com/user-attachments/assets/2b78786a-c76f-4578-ba68-62abda0ac383">

3. Change the width of the screen and check that it looks like this (check the width of the cards and the alignment of the title:

![image](https://github.com/user-attachments/assets/d30ee298-519f-43d6-9b13-6eec447e1784)
![image](https://github.com/user-attachments/assets/9f63081e-efc1-4572-9d4b-81cf6d91606a)
![localhost_ (5)](https://github.com/user-attachments/assets/de4f9038-f092-4999-9ecb-b20e3dfc0928)
![localhost_ (7)](https://github.com/user-attachments/assets/6ae2197d-0d9e-42b9-abc5-376b7878c12e)

4. Check that the Welcome and How do I boxes are centered in mobile views and both have the same width

### Requires New Config

### Requires New Content

### Validation Steps
## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
5. Find the commit of this pull request.
6. Build and deploy the changes.
7. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
